### ISSUE TITLE
Learning Problem Notes 1 Problem 5 - Centering the Data

### ISSUE THREAD

**Issue by avagreyyy4** [mikeizbicki/cmc-csci145-math166#173]
Hi! I have attempted Notes 1, Problem 5, Section 2, where Alice proposes "centering" the data. Intuitively it makes sense for the convergence to be faster since R is getting smaller and seemingly there is not affect on rho since the shift isn't bringing the points closer to w*. However, I am a bit confused on how to show this mathematically. Is it that the only change is that R' = R - $\bar{X}$?

**Comment by mikeizbicki** [comment]
You're correct.  The value of rho stays the same, and wstar rotates (which doesn't change the magnitude).
